solution.rb

```diff
@@ -0,0 +1,21 @@
+def move_zeroes(nums)
+  for i in (0..nums.length)
+    if nums[i] == 0
+        nums << nums.slice!(i, 1).first
+    end
+  end
+  nums
+end
+	
+
+def two_sum(numbers, target)
+	final_hash = {}
```

Comment:
You don't use this variable. Might be confusing.

Reply:
Ok fixed it.